code/_globalvars/lists/objects.dm

```diff
@@ -14,4 +14,5 @@ var/global/list/chemical_reagents_list				//list of all /datum/reagent datums in
 var/global/list/surgeries_list = list()				//list of all surgeries by name, associated with their path.
 var/global/list/table_recipes = list()				//list of all table craft recipes
 var/global/list/rcd_list = list()					//list of Rapid Construction Devices.
-var/global/list/apcs_list = list()					//list of all Area Power Controller machines, seperate from machines for powernet speeeeeeed.
\ No newline at end of file
+var/global/list/apcs_list = list()					//list of all Area Power Controller machines, seperate from machines for powernet speeeeeeed.
+var/global/list/tracking_implants = list()			//list of all tracking implants to work out what treks everyone are on. Sadly not on lavaworld not implemented...
```

Comment:
ill do a fix PR gonna move tracking implants out of the general implant file to make it clearer anyhow.